MMM-pages.js

```diff
@@ -244,4 +264,42 @@ Module.register('MMM-pages', {
       }, delay);
     }
   },
+
+  /**
+   * Pause or resume the page rotation. If the provided pause value is
+   * set to false, it will resume the rotation. If the requested
+   * state (f.e. "pause" === true) equals the current state, print a warning and do nothing.
+   *
+   * @param {boolean} pause the parameter, if you want to pause or resume.
+   */
+  pauseRotation: function (pause) {
```

Comment:
Not a big fan of this name -- can we change this to be something like `setRotation: function (isRotating)` instead?

Reply:
fixed